spec/osut_tests_spec.rb

```diff
@@ -1289,9 +1289,27 @@ module M
     expect(cc.setTemperatureCalculationRequestedAfterLayerNumber(1)).to be true
     expect(floor.setConstruction(cc)).to be true
 
+    # Test 'fixed interval' schedule. Annual time series - no variation.
+    start  = model.getYearDescription.makeDate(1, 1)
+    inter  = OpenStudio::Time.new(0, 1, 0, 0)
+    values = OpenStudio.createVector(Array.new(8760, 22.78))
+    series = OpenStudio::TimeSeries.new(start, inter, values, "")
```

Comment:
_OSut_ enabled _ScheduleFixedInterval_ checks, e.g.:

- min/max setpoint temperatures
- is space UNCONDITIONED, etc.

Yet no unit test had been in place.